### PR TITLE
Fixed an error reading large files and Modified the data reading mode of LR task

### PR DESCRIPTION
**Pull request by mi-sery**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](https://github.com/primihub/community#development-workflow) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your master!
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] Please provide a readable PR title.

### Description
1.Fixed the error of incomplete reading of four arithmetic operations and missing value tasks when reading large files
2.The data set reading method of LR task is modified: two data sets are required instead of one

### CODE REVIEW

File: src/primihub/algorithm/arithmetic.cc [hunk 1]
```diff
@@ -227,13 +223,14 @@ int ArithmeticExecutor<Dbit>::execute() {
...
         mpc_op_exec_->MPC_Compare(sh_res);
       // reveal
-      if (party_id_ == 0) {
-        i64Matrix tmp;
-        tmp = mpc_op_exec_->reveal(sh_res);
-        for (size_t i = 0; i < tmp.rows(); i++)
-          cmp_res_.emplace_back(static_cast<bool>(tmp(i, 0)));
-      } else {
-        mpc_op_exec_->reveal(sh_res, 0);
+      for (auto party : parties_) {
```
Comment (phoenix20162016) [inline src/primihub/algorithm/arithmetic.cc:226]: using const auto& instead of  auto to avoid copy

File: src/primihub/algorithm/arithmetic.cc [hunk 2]
```diff
@@ -272,8 +270,16 @@ int ArithmeticExecutor<Dbit>::finishPartyComm(void) {
   return 0;
 }
 
-template <Decimal Dbit>
-int ArithmeticExecutor<Dbit>::saveModel(void) {
+template <Decimal Dbit> int ArithmeticExecutor<Dbit>::saveModel(void) {
+  bool is_reveal = false;
+  for (auto party : parties_) {
+    if (party == party_id_) {
+      is_reveal = true;
```
Comment (phoenix20162016) [inline src/primihub/algorithm/arithmetic.cc:277]: break loop if set true

File: src/primihub/algorithm/arithmetic.cc (hunk 1 above)
Comment (mi-sery) [reply src/primihub/algorithm/arithmetic.cc]: OK, modification completed

File: src/primihub/algorithm/arithmetic.cc (hunk 2 above)
Comment (mi-sery) [reply src/primihub/algorithm/arithmetic.cc:277]: OK, modification completed

Comment (mi-sery) [reply src/primihub/algorithm/arithmetic.cc:277]: > break loop if set true

OK, modification completed

File: src/primihub/algorithm/arithmetic.cc (hunk 1 above)
Comment (mi-sery) [reply src/primihub/algorithm/arithmetic.cc]: > using const auto& instead of auto to avoid copy

OK, modification completed

**Pull request merged**